src/components/EllipsisLoader/index.js

```diff
@@ -1,37 +1,17 @@
 /** @format */
 
 import React from 'react';
-import PropTypes from 'prop-types';
 import './scss/Styles.scss';
 
-const EllipsisLoader = ({ dotColor }) => (
-  <div className="loader">
-    <div className="loader__lds-ellipsis">
-      <div
-        className="loader__lds-ellipsis__dot"
-        style={{ backgroundColor: dotColor }}
-      />
-      <div
-        className="loader__lds-ellipsis__dot"
-        style={{ backgroundColor: dotColor }}
-      />
-      <div
-        className="loader__lds-ellipsis__dot"
-        style={{ backgroundColor: dotColor }}
-      />
-      <div
-        className="loader__lds-ellipsis__dot"
-        style={{ backgroundColor: dotColor }}
-      />
+const EllipsisLoader = ({ baseClassName = 'loader' }) => (
+  <div className={baseClassName}>
+    <div className={`${baseClassName}__lds-ellipsis`}>
```

Comment:
it feel like the class name `__lds-ellispsis` is a bit of a copy pasta - plz make it shorter and more concise - better naming of classes and components is one of the factors we should be refactoring when transferring to he Library

Reply:
done